### PR TITLE
EPRS: 6.2.18 release

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Addressed https://enterprisedb.atlassian.net/browse/XDB-1345

### CODE REVIEW

**Review by anwar-sahib — APPROVED** [review]
Note along with SQL query added.

**Review by anwar-sahib — DISMISSED** [review]
Command and text corrected.

File: product_docs/docs/eprs/6.2/08_xdb_cli/03_xdb_cli_commands/25_set_pdn_node.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ Examples
 The following example sets the primary node with publication database `ID 9` as the primary definition node.
 
 ```text
-$ java -jar edb-repcli.jar -setasPDN 9 -repsvrfile ~/pubsvrfile.prop
-Updating the database node to be promoted as the new PDN node.
-The database has been successfully promoted as the new PDN node.
+$ java -jar edb-repcli.jar -setasMDN 9 -repsvrfile ~/pubsvrfile.prop
```
Comment (anwar-sahib) [inline product_docs/docs/eprs/6.2/08_xdb_cli/03_xdb_cli_commands/25_set_pdn_node.mdx:33]: Please change the command to 'setasmdn'. Rep cli commands are all lower case for simplicity.

**Comment by drothery-edb** [comment]
@mlewandowski-edb I've already merged this, but if you have any comments I can do another build.

**Pull request merged**